templates/feed.hbs

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom" xml:lang="en">
+    <generator uri="https://blog.rust-lang.org/" version="0.1.0">Rust Blog</generator>
+    <link href="https://blog.rust-lang.org/feed.xml" rel="self" type="application/atom+xml" />
```

Comment:
The URL, title and subtitle are all hard coded into the template, I could change this in the `render_feed` method if desired.

Reply:
I think that for now, it being hardcoded is perfectly file. It's unclear if this code will ever be used for anything but the blog.